2.evaluate_model/nbconverted/genotype_classification_evaluation.py

```diff
@@ -52,11 +53,11 @@
 # In[3]:
 
 
-data_path = pathlib.Path(f"{root_dir}/1.train_models/classify_genotypes/data")
+data_path = pathlib.Path(f"{root_dir}/1.train_models/data")
 
-evaldf = pd.read_parquet(f"{data_path}/nf1_model_pre_evaluation_results.parquet")
-model = load(f"{data_path}/trained_nf1_model.joblib")
-le = load(f"{data_path}/trained_nf1_model_label_encoder.joblib")
+evaldf = pd.read_parquet(f"{data_path}/nf1_model_pre_evaluation_results_qc.parquet")
```

Comment:
is this notebook run twice? (or only after qc?) would a user know to change the `_qc` suffix to evaluate the non-qc data?